apps/frontend/src/components/inputs/PasswordInput/PasswordInput.tsx

```diff
@@ -0,0 +1,52 @@
+import { EyeIcon, EyeSlashIcon } from '@heroicons/react/24/solid';
+import clsx from 'clsx';
+import React, { useState } from 'react';
+
+import type { PasswordInputProps } from './types';
+
+export function PasswordInput({
+  field,
+  form: { touched, errors },
+  label,
+  className,
+  ...props
+}: PasswordInputProps) {
+  const [showPassword, setShowPassword] = useState(false);
+  const toggleShow = () => {
+    setShowPassword((prev) => !prev);
+  };
+
+  return (
+    <div>
+      {label ? <span className="label p-0">{label}</span> : null}
```

Comment:
Let's add a couple of additional props to the component for more flexibility. Add `lableProps` (and spread them into this `label`). And `errorProps`  (and spread them into the error `label`)

Reply:
It would be better to replace this `span` tg with a `label`